components/Events/Filters.js

```diff
@@ -0,0 +1,222 @@
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+export default function filters(props) {
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+  const [chosenComms, setChosenComms] = useState({
+    'studio': false,
+    'icpc': false,
+    'design': false,
+    'cyber': false,
+    'teach-la': false,
+    'w': false,
+    'ai': false,
+    'hack': false,
+    'board': false,
+  });
+  const [chosenDays, setChosenDays] = useState({
+    Sunday: false,
+    Monday: false,
+    Tuesday: false,
+    Wednesday: false,
+    Thursday: false,
+    Friday: false,
+    Saturday: false,
+  });
+  const [chosenTimes, setChosenTimes] = useState({
+    fromTime: '',
+    toTime: '',
```

Comment:
nit, no need to change if you don't want to: 
```suggestion
    from: '',
    to: '',
```
Since `Time` is redundant. It's not a big deal so I don't mind leaving as is